lib/platform.py

```diff
@@ -0,0 +1,270 @@
+import time
+import subprocess
+from subprocess import call
+import os
+import filecmp
+from shutil import copyfile
+
+class Hotspot:
+    def __init__(self, platform):
+        self.hotspot_script_time = 0
+        self.time_without_wifi = 0
+        self.last_wifi_check_time = 0
+        self.is_hostapd_installed = platform.is_package_installed("hostapd")
+
+class Platform_null():
+    def __getattr__(self, name):
+        return self.pass_func
+
+    def pass_func(self, *args, **kwargs):
+        pass
+
+    def get_current_connections(self):
+        return False, "Platform disabled", ""
+
+class PlatformRasp():
+    def copy_connectall_script(self):
+        # make sure connectall.py file exists and is updated
+        if not os.path.exists('/usr/local/bin/connectall.py') or \
+                filecmp.cmp('/usr/local/bin/connectall.py', 'lib/connectall.py') is not True:
+            print("connectall.py script is outdated, updating...")
+            copyfile('lib/connectall.py', '/usr/local/bin/connectall.py')
```

Comment:
It tries to copy "connectall.py" into directory that might not exist on other systems.